src/libraries/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngKey.EC.cs

```diff
@@ -26,7 +26,7 @@ internal static bool IsECNamedCurve(string algorithm)
                 algorithm == CngAlgorithm.ECDsa.Algorithm);
         }
 
-        internal string GetCurveName(out string oidValue)
+        internal string? GetCurveName(out string? oidValue)
```

Comment:
I don't think null can be returned from here.

Reply:
Why is that? Are lookups on that particular key always guaranteed to return a result?

Reply:
When IsECNamedCurve is true the key has to have the curve name property. When it's false we provide a value or throw.

Reply:
FWIW all calls to `GetCurveName` run null checks on the result. I'd be inclined to keep as-is, given it's internal.

Reply:
Alrighty, seems fair.

It's probably a bit over-defensive; like CngKey.AlgorithmGroup having a defensive nullcheck in it... I can't imagine what it means for a CNG key to not say what algorithm group it belongs to... but I can't argue with what the code is doing, vs what I'd expect it to do.